packages/bruno-js/src/bru.js

```diff
@@ -1,7 +1,7 @@
 const Handlebars = require('handlebars');
 const { cloneDeep } = require('lodash');
 
-const envVariableNameRegex = /^(?!\d)[\w-]*$/;
+const envVariableNameRegex = /^(?!\d)[\w-.]*$/;
```

Comment:
Why don't we import this from `packages/bruno-app/src/utils/common/regex.js`?

Reply:
Great point, I agree and I'd also prefer to import it from one place instead of duplicating it. I wasn't sure whether to refactor it in this PR or a separate PR (not sure how it would be preferred in this project).

In this specific case, the `bruno-app` package is the standalone Next.js app, whereas the `bruno-js` package is a library used by the other packages in the codebase, so I think we can export the regex from `bruno-js` (or maybe another common/util package).

Reply:
I'll refactor it after all, thanks!